### ISSUE TITLE
Does this work on Windows 8.1?

### ISSUE THREAD

**Issue by awayken** [yahoo/reginabox#1]
I ran the `npm` command on Windows 8.1, and it would not build. I'm curious if anyone has tried installing this on Windows, or if there are known issues in Windows. If you want, I can start pasting in error messages.

**Comment by bengl** [comment]
yes, error messages would be useful.

**Comment by awayken** [comment]
@bengl I don't know how I missed your reply for so long!

I just tried again to install `reginabox` and here's the tail end of my error messages. Let me know if I should paste more information (or the entire npm-debug.log) somewhere for viewing.

```
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (D:\Program Files\iojs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "D:\\Program Files\\iojs\\node.exe" "D:\\Program Files\\iojs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\mrausch\AppData\Roaming\npm\node_modules\reginabox\node_modules\mdns
gyp ERR! node -v v2.0.2
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm WARN engine follow@0.11.4: wanted: {"node":"0.10.x || 0.8.x"} (current: {"node":"2.0.2","npm":"2.9.0"})
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "D:\\Program Files\\iojs\\node.exe" "D:\\Program Files\\iojs\\node_modules\\npm\\bin\\npm-cli.js" "i" "-g" "reginabox"
npm ERR! node v2.0.2
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE

npm ERR! mdns@2.2.9 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mdns@2.2.9 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the mdns package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls mdns
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\Sites\npm-debug.log
```

**Comment by bengl** [comment]
Things are a little trickier on Windows, but you still should be able to get this going. It's failing on installing `node-mdns`. You're going to need Apple's "[Bonjour SDK for Windows](https://developer.apple.com/downloads/index.action?q=Bonjour%20SDK%20for%20Windows)" (You'll need an Apple Developer account). See the [node-mdns installation instructions](https://github.com/agnat/node_mdns#installation).

If that doesn't fix things for you, please dump the entire `npm-debug.log` into a gist or pastebin or something, and I'll have a closer look.

**Comment by awayken** [comment]
@bengl You're right. That is where I'm having issues. I've started down the `node_mdns` path and that cleared up some errors, but there are more. At this point, I'm not sure all this work is worth it, considering I just happened upon this repo and wanted to play around a little bit.

I've put in a pull request (#3) to add a little caveat language for future Windows adventurers but I consider this issue closed. I'll let some future, enterprising Windows developer take it up from here.
